### ISSUE TITLE
Tab sizes

### ISSUE THREAD

**Issue by Borf** [Requarks/wiki#136]
### Actual behavior
The tab sizes in the text editor are different from the tabsizes in the generated html. This is mainly an issue in code blocks, as these use a fixed width font.

### Expected behavior
Expected is the that a piece of aligned code looks the same in the editor as in the generated html

### Steps to reproduce the behavior
Use the following code
```
	Just use tabs	and it won't align
				but it won't
```

**Comment by NGPixel** [comment]
The issue is when you paste code code into the editor that contains tabs. The editor will not convert them. If you write code directly into the editor and press tab, it will be converted to spaces correctly.

I'll look into the issue but as a general rule, you should avoid using hard tabs for code. They create a myriad of problems. All code editors supports soft tabs.

**Comment by Borf** [comment]
Sorry, but I write all my code using hard tabs instead of soft tabs, editors still don't work well with soft tabs in my opinion (but that's a different discussion). 
I'd think you can control the tab-size using https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size on the rendered page, and setOption("tabSize", xxx) on the ACE editor

**Comment by NGPixel** [comment]
Yeah I added the code already. The "Insert Code" modal (using Ace) and the actual rendered result are now the same, but the markdown editor doesn't display the tabs correctly (using CodeMirror). Some more testing needed...
